src/cascade/input_data/db/demographics.py

```diff
@@ -30,20 +31,38 @@ def get_locations(execution_context):
     ]
 
 
-def age_groups_to_ranges(execution_context, data):
+def age_groups_to_ranges(execution_context, data, keep_age_group_id=False):
     groups = get_age_groups(execution_context)
-    with_group = data.merge(groups, on="age_group_id").drop(columns="age_group_id")
+    with_group = data.merge(groups, on="age_group_id")
+    if not keep_age_group_id:
+        with_group = with_group.drop(columns="age_group_id")
     return with_group.rename(columns={"age_group_years_start": "age_lower", "age_group_years_end": "age_upper"})
 
 
-def get_all_age_spans(execution_context):
+def get_mean_years(execution_context):
```

Comment:
This doesn't really bother me, but I feel a little question arising (before reading the docstring), does mean_years mean something about a span like 2005-2010, though years plural, sounds like it doesn't, or does it mean something about an age group like 5-10?  =7.5 years

Reply:
Neither. It is the average age that you expect someone within this age group to have minus the starting age of the group. If you can think of a better name for that, I'm all ears because I don't have a good one.

Reply:
Alec, did you get rid of the age spans call? I'm sure we will need it to know the start and finish of each span. Maybe, if and when it's surely necessary, then it's easy to put in.

Reply:
Look down, the diff is just confused.